make_ports_public.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Why is there code related to codespaces here? That should be in a different PR to make sure that we have a shorter PR to review here.